{
  "uid" : "d72494c7899be651",
  "name" : "test_02_file[file_name_info1]",
  "fullName" : "TestCase.test_vue.Test_Login_Vue#test_02_file",
  "historyId" : "d7f73096832c045b1e34718b57c5e48a",
  "time" : {
    "start" : 1647331833447,
    "stop" : 1647331833447,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
  "statusTrace" : "self = <class 'TestCase.test_vue.Test_Login_Vue'>\n\n    def setup_class(self) -> None:\n        # 打开浏览器\n        global driver\n>       self.driver = webdriver.Chrome()\n\nTestCase/test_vue.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nvenv/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py:70: in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\nvenv/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py:90: in __init__\n    self.service.start()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.chrome.service.Service object at 0x7fbc4db24ac0>\n\n    def start(self):\n        \"\"\"\n        Starts the Service.\n    \n        :Exceptions:\n         - WebDriverException : Raised either when it can't start the service\n           or when it can't connect to the service\n        \"\"\"\n        try:\n            cmd = [self.path]\n            cmd.extend(self.command_line_args())\n            self.process = subprocess.Popen(cmd, env=self.env,\n                                            close_fds=system() != 'Windows',\n                                            stdout=self.log_file,\n                                            stderr=self.log_file,\n                                            stdin=PIPE,\n                                            creationflags=self.creationflags)\n        except TypeError:\n            raise\n        except OSError as err:\n            if err.errno == errno.ENOENT:\n                raise WebDriverException(\n                    \"'%s' executable needs to be in PATH. %s\" % (\n                        os.path.basename(self.path), self.start_error_message)\n                )\n            elif err.errno == errno.EACCES:\n                raise WebDriverException(\n                    \"'%s' executable may have wrong permissions. %s\" % (\n                        os.path.basename(self.path), self.start_error_message)\n                )\n            else:\n                raise\n        except Exception as e:\n            raise WebDriverException(\n                \"The executable %s needs to be available in the path. %s\\n%s\" %\n                (os.path.basename(self.path), self.start_error_message, str(e)))\n        count = 0\n        while True:\n            self.assert_process_still_running()\n            if self.is_connectable():\n                break\n    \n            count += 1\n            sleep(0.5)\n            if count == 60:\n>               raise WebDriverException(\"Can not connect to the Service %s\" % self.path)\nE               selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/common/service.py:105: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 7,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1647331802738,
      "stop" : 1647331833334,
      "duration" : 30596
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver\n\n",
    "statusTrace" : "  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/TestCase/test_vue.py\", line 23, in setup_class\n    self.driver = webdriver.Chrome()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py\", line 70, in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py\", line 90, in __init__\n    self.service.start()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/common/service.py\", line 105, in start\n    raise WebDriverException(\"Can not connect to the Service %s\" % self.path)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "TestCase"
  }, {
    "name" : "suite",
    "value" : "test_vue"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login_Vue"
  }, {
    "name" : "host",
    "value" : "YZQ.local"
  }, {
    "name" : "thread",
    "value" : "9912-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCase.test_vue"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "file_name_info",
    "value" : "[2, '输入正确的文件名，提示下载成功', 10000000000, 'y', None, None]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "a3163cc59418eabc",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331748439,
        "stop" : 1647331748439,
        "duration" : 0
      }
    }, {
      "uid" : "93f8a9cb33d0df5a",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331604430,
        "stop" : 1647331604430,
        "duration" : 0
      }
    }, {
      "uid" : "232a45cad0b23198",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver",
      "time" : {
        "start" : 1647331512608,
        "stop" : 1647331512608,
        "duration" : 0
      }
    }, {
      "uid" : "f1126ce7743006f2",
      "status" : "broken",
      "statusDetails" : "ValueError: too many values to unpack (expected 5)",
      "time" : {
        "start" : 1647309470969,
        "stop" : 1647309470969,
        "duration" : 0
      }
    }, {
      "uid" : "87d8e6449f25ca86",
      "status" : "broken",
      "statusDetails" : "ValueError: too many values to unpack (expected 5)",
      "time" : {
        "start" : 1647309360705,
        "stop" : 1647309360705,
        "duration" : 0
      }
    }, {
      "uid" : "4503aa397fcdb838",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='app']/div/form/button\"}\n  (Session info: chrome=99.0.4844.51)\nStacktrace:\n0   chromedriver                        0x0000000103ec2ee9 chromedriver + 5013225\n1   chromedriver                        0x0000000103e4e1d3 chromedriver + 4534739\n2   chromedriver                        0x0000000103a24a68 chromedriver + 170600\n3   chromedriver                        0x0000000103a59152 chromedriver + 385362\n4   chromedriver                        0x0000000103a59311 chromedriver + 385809\n5   chromedriver                        0x0000000103a8b324 chromedriver + 590628\n6   chromedriver                        0x0000000103a7680d chromedriver + 505869\n7   chromedriver                        0x0000000103a89074 chromedriver + 581748\n8   chromedriver                        0x0000000103a766d3 chromedriver + 505555\n9   chromedriver                        0x0000000103a4c76e chromedriver + 333678\n10  chromedriver                        0x0000000103a4d745 chromedriver + 337733\n11  chromedriver                        0x0000000103e7eefe chromedriver + 4734718\n12  chromedriver                        0x0000000103e98a19 chromedriver + 4839961\n13  chromedriver                        0x0000000103e9e1c8 chromedriver + 4862408\n14  chromedriver                        0x0000000103e993aa chromedriver + 4842410\n15  chromedriver                        0x0000000103e73a01 chromedriver + 4688385\n16  chromedriver                        0x0000000103eb4538 chromedriver + 4953400\n17  chromedriver                        0x0000000103eb46c1 chromedriver + 4953793\n18  chromedriver                        0x0000000103eca225 chromedriver + 5042725\n19  libsystem_pthread.dylib             0x00007fff203a98fc _pthread_start + 224\n20  libsystem_pthread.dylib             0x00007fff203a5443 thread_start + 15",
      "time" : {
        "start" : 1646293856465,
        "stop" : 1646293859440,
        "duration" : 2975
      }
    }, {
      "uid" : "ab7a0e73e1f49071",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='app']/div/form/button\"}\n  (Session info: chrome=99.0.4844.51)\nStacktrace:\n0   chromedriver                        0x000000010a308ee9 chromedriver + 5013225\n1   chromedriver                        0x000000010a2941d3 chromedriver + 4534739\n2   chromedriver                        0x0000000109e6aa68 chromedriver + 170600\n3   chromedriver                        0x0000000109e9f152 chromedriver + 385362\n4   chromedriver                        0x0000000109e9f311 chromedriver + 385809\n5   chromedriver                        0x0000000109ed1324 chromedriver + 590628\n6   chromedriver                        0x0000000109ebc80d chromedriver + 505869\n7   chromedriver                        0x0000000109ecf074 chromedriver + 581748\n8   chromedriver                        0x0000000109ebc6d3 chromedriver + 505555\n9   chromedriver                        0x0000000109e9276e chromedriver + 333678\n10  chromedriver                        0x0000000109e93745 chromedriver + 337733\n11  chromedriver                        0x000000010a2c4efe chromedriver + 4734718\n12  chromedriver                        0x000000010a2dea19 chromedriver + 4839961\n13  chromedriver                        0x000000010a2e41c8 chromedriver + 4862408\n14  chromedriver                        0x000000010a2df3aa chromedriver + 4842410\n15  chromedriver                        0x000000010a2b9a01 chromedriver + 4688385\n16  chromedriver                        0x000000010a2fa538 chromedriver + 4953400\n17  chromedriver                        0x000000010a2fa6c1 chromedriver + 4953793\n18  chromedriver                        0x000000010a310225 chromedriver + 5042725\n19  libsystem_pthread.dylib             0x00007fff203a98fc _pthread_start + 224\n20  libsystem_pthread.dylib             0x00007fff203a5443 thread_start + 15",
      "time" : {
        "start" : 1646293809711,
        "stop" : 1646293810957,
        "duration" : 1246
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "d72494c7899be651.json",
  "parameterValues" : [ "[2, '输入正确的文件名，提示下载成功', 10000000000, 'y', None, None]" ]
}